common/src/Model/CompilationTask.h

```diff
@@ -71,6 +71,26 @@ namespace TrenchBroom {
             deleteCopyAndMove(CompilationExportMap)
         };
 
+        class CompilationExportObj : public CompilationTask {
```

Comment:
Is it not possible to parameterize the `CompilationExportMap` task?

Reply:
Not sure I understand, do you mean adding a comment explaining what `CompilationExportObj` does?

Reply:
Ahh, you mean adding a boolean to the `CompilationExportMap` task. That should be possible and maybe a bit simpler? I wasn't sure how `CompilationExportMap` might change in the future though.

Reply:
Not necessarily a boolean, but a more generic approach, since we may have multiple export formats in the future. The problem is that different formats may require different UIs, too, so maybe we should leave them separate for now so that we can have different UIs for each export task. We can always refactor in the future.